From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Martijn Muijsers <martijnmuijsers@live.nl>
Date: Wed, 12 Jul 2023 22:14:25 +0200
Subject: [PATCH] Modifiable Bukkit enums - Inject runtime versions - Depend on
 ByteBuddy

License: Fiddle Public License Agreement 1.0 (included in license/FPL-1.0.txt)
Fiddle - https://fiddlemc.org

diff --git a/build.gradle.kts b/build.gradle.kts
index 22f44b1d5d390d0960d97e8767c9988d4330995f..e9c218fca68153768a76546bf7eaaf0d43a7d304 100644
--- a/build.gradle.kts
+++ b/build.gradle.kts
@@ -65,6 +65,7 @@ dependencies {
     testImplementation("org.hamcrest:hamcrest-library:1.3")
 
     implementation("io.netty:netty-all:4.1.87.Final"); // Paper - Bump netty
+    implementation("net.bytebuddy:byte-buddy:1.14.5") // Fiddle - modifiable Bukkit enums - inject runtime versions - depend on ByteBuddy
 }
 
 val craftbukkitPackageVersion = "1_20_R1" // Paper
